code/index.html

```diff
@@ -2,17 +2,186 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project Name</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fake Agency</title>
+    <link rel="icon" type="image/x-icon" href="favicon.ico">
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=EB+Garamond:ital@1&family=Fira+Sans:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
   </head>
+
   <body>
     <header>
-      <!-- If you want a header with a title, logo and a navbar, here is where you should add that. -->
+    <div class="hamburger">
+        <div></div>
+        <div></div>
+        <div></div>
+    </div>
+    <div class="header-left">
+      <a href="./index.html"> <img src="logo.png" alt="Logo Fake Agency"></a>
+    </div>
+    <div class="filler"></div>
+    <div class="header-right">
+      <nav>
+          <a href="#"> work </a>
+          <a href="#"> services </a>
+          <a href="#"> about </a>
+          <a href="#"> contact </a>
+      </nav>
+    </div>
     </header>
 
-    <main class="">
-      <!-- The main container to use as the flex parent for your card layout, the main content of the site should be in here.  -->
+    <main class="main-container"> 
+      
+      <section class="hero-container">
+        <h1>We <em>humanise</em> brands.<br> We <em>connect</em> them to culture. <br> We make content that <em>converts</em>.</h1>
+      </section>
+      
+      <section class="work-container">
+        <h2>Our Work</h2>
+      <section class="image-gallery">
```

Comment:
Nice image grid!

Reply:
Would be nice to keep all images in a folder so that they don't clutter the code folder :)